src/recorder.js

```diff
@@ -0,0 +1,17 @@
+export default class Recorder {
+    record(span) {
+        console.log(JSON.stringify({ // eslint-disable-line
```

Comment:
something like a `DebuggingRecorder` would be great, but the default Recorder should do nothing at all.